.gitignore

```diff
@@ -29,3 +29,7 @@ debug.test
 # OSX auto-generated file
 **/.DS_Store
 *.env
+
+# docker builds
+docker
```

Comment:
I think this can be dropped from gitignore now that you moved the generated content to output dir